manifests/params.pp

```diff
@@ -54,5 +55,8 @@
     }
   }
 
+  if $default_package_ensure == undef {
+    $package_ensure = $default_package_ensure
+  }
```

Comment:
I don't htink this block does what you want it to.

Since there aren't any special cases would prefer to just set `$package_ensure` to 'installed'